.gitpod.yml

```diff
@@ -35,10 +29,13 @@ ports:
 # some useful extensions to have
 vscode:
   extensions:
-    - eamodio.gitlens@10.2.3:CRhSs/nkGIS3K5yCBlhVTg==
-    - njpwerner.autodocstring@0.5.4:TKRAH5+nBrBnq+ysTYA86Q==
-    - lextudio.restructuredtext@141.0.0:JobzqAU1ylF4bDdKqnrpOg==
-    - ritwickdey.liveserver@5.6.1:5Mhl1RgVVDcIym163x8r+w==
+    - eamodio.gitlens
```

Comment:
When trying to use Gitlens in your snapshot image, I get:
```
Activating extension 'GitLens — Git supercharged' failed: Cannot read property 'fsPath' of undefined
```

Reply:
🤔  that is an issue I had never encountered - will look into it

Reply:
Solved now - the extension loader needed restarting from VSCode